src/pages/guidelines/2x-grid/implementation.mdx

```diff
@@ -1343,6 +1344,42 @@ Image size can change ratios when the breakpoint changes.
   </DoDont>
 </DoDontRow>
 
+## API
+
+There are many ways to make the 2x Grid work for your application using our API.
+The example below illustrates removing visibility of a given component based on
+the maximum width of the `md` breakpoint. Check out the table to learn more
+about our API.
+
+```css
+.component {
+  display: inline;
+
+  /* applies styles inside mixin on md breakpoint or below, like max-width */
+  @include carbon--breakpoint-down('md') {
+    display: none;
+  }
+}
+```
+
+| Name                     | Description                                                                         | Type        | Default       |
+| :----------------------- | :---------------------------------------------------------------------------------- | :---------- | :------------ |
+| `css-grid`               | Generate the CSS for using the CSS Grid                                             | `@mixin`    |               |
+| `$prefix`                | Specify the prefix used for CSS selectors                                           | `String`    | `'cds'`       |
+| `$flex-grid-columns`     | Specify the number of columns for the flex grid                                     | `Number`    | `16`          |
+| `$grid-gutter`           | Specify the gutter of the grid                                                      | `Number`    | `32px (2rem)` |
+| `$grid-gutter-condensed` | Specify the gutter of the condensed grid                                            | `Number`    | `1px`         |
+| `$grid-breakpoints`      | Specify the breakpoints for the grid                                                | `Map`       |               |
+| `breakpoint-next`        | Get the value of the next breakpoint                                                | `@function` |               |
+| `breakpoint-prev`        | Get the value of the previous breakpoint                                            | `@function` |               |
+| `is-smallest-breakpoint` | Check to see if the given breakpoint is the smallest breakpoint                     | `@function` |               |
+| `is-largest-breakpoint`  | Check to see if the given breakpoint is the largest breakpoint                      | `@function` |               |
+| `breakpoint-up`          | Generate a media query from the width of the given breakpoint to infinity           | `@mixin`    |               |
+| `breakpoint-down`        | Generate a media query that applies below the maximum width of the given breakpoint | `@mixin`    |               |
+| `breakpoint-between`     | Generate a media query for the range between the lower and upper breakpoints        | `@mixin`    |               |
+| `largest-breakpoint`     | Generate a media query for the largest breakpoint                                   | `@mixin`    |               |
+| `breakpoint`             | Generate a media query for a given breakpoint, alias of `breakpoint-up`             | `@mixin`    |               |
```

Comment:
Could we instead link to the grid package Sass API docs? 

https://github.com/carbon-design-system/carbon/blob/main/packages/grid/docs/sass.md#api

If we duplicate this here, one of them is bound to become out of date. I don't think we surface full Sass API information anywhere else on the website.

Reply:
I think we do have a resource tile that links to the package readme. I think there's a big need to surface these here as well. We tend to do this with all our elements stuff like spacing, motion sass - https://carbondesignsystem-fq9t0896o-carbon-design-system.vercel.app/guidelines/motion/code but we don't do anything for grid. We don't surface that code anywhere on our website. Totally get the issue about these becoming out of date though. I'm not sure of a workaround for that. How often do we make changes to the grid API?

Reply:
Ok, I think I may have found a middle ground. Let me know what you think.